src/tests/generatekey.cpp

```diff
@@ -1148,6 +1148,7 @@ TEST_F(rnp_tests, test_generated_key_sigs)
         pgp_signature_info_t     ssiginfo = {};
 
         memset(&desc, 0, sizeof(desc));
+        desc.pgp_version = PGP_V4;
```

Comment:
This should be put under ifdef, so we make sure that without setting a version everything works as before.